js/analyzer.js

```diff
@@ -79,15 +84,15 @@ Analyzer.prototype.loadWordlists = function() {
 /**
  * set default variables.
  */
-Analyzer.prototype.setDefaults = function() {
+YoastSEO_Analyzer.prototype.setDefaults = function() {
     //set default variables
-    this.keywordRegex = new RegExp( this.config.keyword );
+    this.keywordRegex = new RegExp( this.config.keywordLowerCase, "ig" );
```

Comment:
Perhaps add inline documentation to explain the regex modifiers ('ig') you're setting, so the code is easier to read